### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by bertdeblock**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,14 +72,13 @@
     "ember-cli-babel": "^7.26.6",
     "ember-cli-dependency-checker": "^3.2.0",
     "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-export-application-global": "^2.0.1",
```
Comment (bertdeblock) [inline package.json:75]: Doesn't work in Ember v5.

File: package.json [hunk 2]
```diff
@@ -72,14 +72,13 @@
...
     "ember-load-initializers": "^2.1.1",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-qunit": "^5.1.4",
     "ember-resolver": "^8.0.2",
     "ember-source": "~3.28.0",
     "ember-source-channel-url": "^3.0.0",
-    "ember-styleguide": "^5.0.0",
+    "ember-styleguide": "^8.4.0",
```
Comment (bertdeblock) [inline package.json:81]: Older version was breaking the build.

File: package.json [hunk 3]
```diff
@@ -106,6 +105,9 @@
   "publishConfig": {
     "registry": "https://registry.npmjs.org"
   },
+  "ember": {
+    "edition": "octane"
```
Comment (bertdeblock) [inline package.json:109]: Required in newer Ember versions.

File: node-tests/colocated-babel-plugin-test.js [hunk 4]
```diff
@@ -30,7 +30,6 @@ describe('ColocatedBabelPlugin', function () {
         stripIndent`
           import Component from '@glimmer/component';
           const __COLOCATED_TEMPLATE__ = 'ok';
-
```
Comment (bertdeblock) [inline node-tests/colocated-babel-plugin-test.js:33]: Deduped the lockfile, which results in newline diffs.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -19,7 +19,7 @@ jobs:
     - uses: actions/checkout@v1
     - uses: actions/setup-node@v1
       with:
-        node-version: 12.x
+        node-version: 16
```
Comment (bertdeblock) [inline .github/workflows/ci.yml:22]: Deduped the lockfile and was getting `engines` errors.

**Review by void-mAlex — APPROVED** [review]

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**